content/en/docs/getting-started-guides/scratch.md

```diff
@@ -493,12 +493,12 @@ You will need to run one or more instances of etcd.
     by durable storage (RAID, GCE PD).
     
     {{< note >}}
-    **Note:** May result in operations outages in case of instance outage.
+    May result in operations outages in case of instance outage.
     {{< /note >}}
```

Comment:
See https://kubernetes.io/docs/home/contribute/style-guide/ - this is correct. Leave the Note in.